app/views/users/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <div class="page-body">
+    <div class="container-xl">
+      <div class="row row-deck row-cards">
+        <div class="col-12">
+          <div class="card">
+            <div class="card-body">
```

Comment:
Majdnem pontosan ugyanez a 7-8 egymásba ágyazott `div` van a `_form` partial-ödben is. Nézz rá légyszi, hogy mind a két helyen biztosan szükség van-e ezekre ahhoz, hogy helyesen jelenjen meg az oldal.